app.json

```diff
@@ -3,7 +3,7 @@
   "stack": "heroku-18",
   "description": "Open data project for small-scale food growers",
   "scripts": {
-    "postdeploy": "bundle exec rails db:migrate db:seed"
+    "postdeploy": "bundle exec rails db:seed"
```

Comment:
Are migrations done elsewhere?

Reply:
yes, the migrations are part of the build pack, which is used on every deploy.
the "postdeploy" is only run once, at first deploy.  Heroku's naming needs work.

Reply:
Works for me :)